<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleReportDlg</class>
 <widget class="QDialog" name="SimpleReportDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расходы времени</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>5</number>
   </property>
   <item alignment="Qt::AlignTop">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>87</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QDateEdit" name="dtFirst">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string> - </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dtSecond">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnBack">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Back</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../TM3res.qrc">
          <normaloff>:/images/new/left-pointing-arrow.png</normaloff>:/images/new/left-pointing-arrow.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnFwd">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Forward</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../TM3res.qrc">
          <normaloff>:/images/new/right-arrow.png</normaloff>:/images/new/right-arrow.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnDay">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Day</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../TM3res.qrc">
          <normaloff>:/images/new/circular-arrow-clock.png</normaloff>:/images/new/circular-arrow-clock.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnWeek">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Week</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../TM3res.qrc">
          <normaloff>:/images/new/calendar-day-15.png</normaloff>:/images/new/calendar-day-15.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnMonth">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Month</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../TM3res.qrc">
          <normaloff>:/images/new/calendar-with-a-clock-time-tools.png</normaloff>:/images/new/calendar-with-a-clock-time-tools.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnYear">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Year</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../TM3res.qrc">
          <normaloff>:/images/new/calendar.png</normaloff>:/images/new/calendar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>76</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>64</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QRadioButton" name="radRate">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Rates</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radCat">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Cats</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="radTag">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Tags</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="editSummary">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>15</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../TM3res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SimpleReportDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>659</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SimpleReportDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>659</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dtFirst</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>SimpleReportDlg</receiver>
   <slot>firstDateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>-78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dtSecond</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>SimpleReportDlg</receiver>
   <slot>secondDateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>-62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBack</sender>
   <signal>clicked()</signal>
   <receiver>SimpleReportDlg</receiver>
   <slot>periodBack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>-49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFwd</sender>
   <signal>clicked()</signal>
   <receiver>SimpleReportDlg</receiver>
   <slot>periodFwd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>-52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDay</sender>
   <signal>clicked()</signal>
   <receiver>SimpleReportDlg</receiver>
   <slot>dailyReport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>-43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnWeek</sender>
   <signal>clicked()</signal>
   <receiver>SimpleReportDlg</receiver>
   <slot>weeklyReport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>-57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMonth</sender>
   <signal>clicked()</signal>
   <receiver>SimpleReportDlg</receiver>
   <slot>monthlyReport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>-31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnYear</sender>
   <signal>clicked()</signal>
   <receiver>SimpleReportDlg</receiver>
   <slot>yearReport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>-90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radCat</sender>
   <signal>clicked(bool)</signal>
   <receiver>SimpleReportDlg</receiver>
   <slot>radCatClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radRate</sender>
   <signal>clicked(bool)</signal>
   <receiver>SimpleReportDlg</receiver>
   <slot>radRateClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>624</x>
     <y>-16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radTag</sender>
   <signal>clicked(bool)</signal>
   <receiver>SimpleReportDlg</receiver>
   <slot>radTagClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>663</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>692</x>
     <y>-28</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>firstDateChanged()</slot>
  <slot>secondDateChanged()</slot>
  <slot>periodBack()</slot>
  <slot>periodFwd()</slot>
  <slot>dailyReport()</slot>
  <slot>weeklyReport()</slot>
  <slot>monthlyReport()</slot>
  <slot>yearReport()</slot>
  <slot>radCatClicked(bool)</slot>
  <slot>radRateClicked(bool)</slot>
  <slot>radTagClicked(bool)</slot>
  <slot>radBudjetClicked(bool)</slot>
  <slot>radWorkTimeClicked(bool)</slot>
 </slots>
</ui>
